### PR TITLE
Update setup-dev-environment-index.adoc

### PR DESCRIPTION
**Pull request by NathalieC**
* complete cards section

### CODE REVIEW

File: modules/ROOT/pages/setup-dev-environment-index.adoc [hunk 1]
```diff
@@ -1,28 +1,45 @@
...
+--
+
+[.card.card-index]
+--
+xref:automating-builds.adoc[[.card-title]#Automating builds# [.card-body.card-content-overflow]#pass:q[Make your project ready for deployment on any environment]#]
+--
+
+[.card.card-index]
+--
+xref:set-up-continuous-integration.adoc[[.card-title]#Set up continuous integration# [.card-body.card-content-overflow]#pass:q[The first environment to deploy into]#]
```
Comment (rbioteau) [inline modules/ROOT/pages/setup-dev-environment-index.adoc]: Does not exist anymore in 2021.2

File: modules/ROOT/pages/setup-dev-environment-index.adoc [hunk 2]
```diff
@@ -1,28 +1,45 @@
...
 --
 
+[.card.card-index]
+--
+xref:migrate-a-svn-repository-to-github.adoc[[.card-title]#How to migrate a SVN repository into a Git repository# [.card-body.card-content-overflow]#pass:q[Because Bonita uses Git]#]
+--
+
+[.card.card-index]
+--
+xref:automating-builds.adoc[[.card-title]#Automating builds# [.card-body.card-content-overflow]#pass:q[Make your project ready for deployment on any environment]#]
```
Comment (rbioteau) [inline modules/ROOT/pages/setup-dev-environment-index.adoc]: Does not exist anymore in 2021.2

**Review by rbioteau — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/setup-dev-environment-index.adoc [hunk 3]
```diff
@@ -1,28 +1,45 @@
 = Set up the Development Environment
-:description: blabla
+:description: A description of all Bonita features that allow the creation of a customized, shared, and automated development environment.
 
-bla bla
+A description of all Bonita features that allow the creation of a customized, shared, and automated development environment.
```
Comment (alachambre) [inline modules/ROOT/pages/setup-dev-environment-index.adoc]:
```suggestion
{description}
```

File: modules/ROOT/pages/setup-dev-environment-index.adoc [hunk 4]
```diff
@@ -1,28 +1,45 @@
...
 [.card.card-index]
 --
-xref:set-up-continuous-integration.adoc[[.card-title]#Set up continuous integration# [.card-body.card-content-overflow]#pass:q[]#]
+xref:workspaces-and-repositories.adoc[[.card-title]#Workspaces and projects# [.card-body.card-content-overflow]#pass:q[The structure of all the content you create]#]
 --
 
 [.card.card-index]
 --
-xref:automating-builds.adoc[[.card-title]#Automating builds# [.card-body.card-content-overflow]#pass:q[]#]
+xref:shared-project.adoc[[.card-title]#Set up Bonita Studio with GitHub# [.card-body.card-content-overflow]#pass:q[Share and version your projects]#]
```
Comment (alachambre) [inline modules/ROOT/pages/setup-dev-environment-index.adoc]: Git != Github 

```suggestion
xref:shared-project.adoc[[.card-title]#Set up Bonita Studio with Git# [.card-body.card-content-overflow]#pass:q[Share and version your projects]#]
```

File: modules/ROOT/pages/setup-dev-environment-index.adoc [hunk 5]
```diff
@@ -1,28 +1,45 @@
...
 --
 
+[.card.card-index]
+--
+xref:migrate-a-svn-repository-to-github.adoc[[.card-title]#How to migrate a SVN repository into a Git repository# [.card-body.card-content-overflow]#pass:q[Because Bonita uses Git]#]
```
Comment (alachambre) [inline modules/ROOT/pages/setup-dev-environment-index.adoc]:
```suggestion
xref:migrate-a-svn-repository-to-github.adoc[[.card-title]#How to migrate a SVN repository into a Git repository# [.card-body.card-content-overflow]#pass:q[Manual to migrate from SVN to Git]#]
```

File: modules/ROOT/pages/setup-dev-environment-index.adoc [hunk 6]
```diff
@@ -1,28 +1,45 @@
...
+[.card.card-index]
+--
+xref:automating-builds.adoc[[.card-title]#Automating builds# [.card-body.card-content-overflow]#pass:q[Make your project ready for deployment on any environment]#]
+--
+
+[.card.card-index]
+--
+xref:set-up-continuous-integration.adoc[[.card-title]#Set up continuous integration# [.card-body.card-content-overflow]#pass:q[The first environment to deploy into]#]
+--
+
```
Comment (alachambre) [inline modules/ROOT/pages/setup-dev-environment-index.adoc]:
```suggestion 
```

File: modules/ROOT/pages/setup-dev-environment-index.adoc [hunk 7]
```diff
@@ -1,28 +1,45 @@
...
+
```
Comment (alachambre) [inline modules/ROOT/pages/setup-dev-environment-index.adoc]:
```suggestion
```

File: modules/ROOT/pages/setup-dev-environment-index.adoc [hunk 8]
```diff
@@ -1,28 +1,45 @@
...
+
```
Comment (alachambre) [inline modules/ROOT/pages/setup-dev-environment-index.adoc]:
```suggestion
```

**Review by alachambre — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/setup-dev-environment-index.adoc (hunk 2 above)
Comment (NathalieC) [reply modules/ROOT/pages/setup-dev-environment-index.adoc]: What do you mean? 
That we do not want to talk about that in 2021.2 ?
Or that I have lost the pages between 2021.1 and 2021.2 ?

File: modules/ROOT/pages/setup-dev-environment-index.adoc (hunk 1 above)
Comment (NathalieC) [reply modules/ROOT/pages/setup-dev-environment-index.adoc]: What do you mean?
That we do not want to talk about that in 2021.2 ?
Or that I have lost the pages between 2021.1 and 2021.2 ?

File: modules/ROOT/pages/setup-dev-environment-index.adoc (hunk 5 above)
Comment (NathalieC) [reply modules/ROOT/pages/setup-dev-environment-index.adoc]: made a change here

File: modules/ROOT/pages/setup-dev-environment-index.adoc (hunk 2 above)
Comment (rbioteau) [reply modules/ROOT/pages/setup-dev-environment-index.adoc]: As the Workspace API has been deleted in 2021.2, those article were not relevant anymore

File: modules/ROOT/pages/setup-dev-environment-index.adoc (hunk 1 above)
Comment (rbioteau) [reply modules/ROOT/pages/setup-dev-environment-index.adoc]: As the Workspace API has been deleted in 2021.2, those article were not relevant anymore

**Pull request merged**